src/senaite/storage/browser/templates/store_container.pt

```diff
@@ -79,7 +75,7 @@
                         {"columnName": "getClientSampleID", "align": "left", "label": "CSID", "width": "15"},
                         {"columnName": "getSampleTypeTitle", "align": "left", "label": "Sample Type", "width": "70"},
                         {"columnName": "UID", "hidden": true}],
-                      "search_fields": ["getId"],
+                      "search_fields": ["SearchableText"],
```

Comment:
@tumiemosweu , would you mind to do the search for index `sortable_title` (only contains the sample id, but not additional information) or by `listing_searchable_text` (contains sample id with additional info from the sample, such as ClientID, etc.) instead of by `SearchableText`. This `SearchableText` index is no longer available in newer versions of this catalog.

Reply:
ok, I will update accordingly. Thank you.